tests/e2e/pkg/tester/kubectl.go

```diff
@@ -83,9 +83,14 @@ func (t *Tester) extractBinaries(downloadPath string) (string, error) {
 	}
 	tarReader := tar.NewReader(gzf)
 
-	kubectlDir, err := os.MkdirTemp("", "kubectl")
-	if err != nil {
-		return "", err
+	var kubectlDir string
+	if dir, ok := os.LookupEnv("KUBETEST2_RUN_DIR"); ok {
```

Comment:
Annoying that rundir is entirely private to the ginkgo tester (which we compose ours of). With this, there will be different behavior if `--rundir` is set vs this variable. Maybe we should shadow `rundir` and do a similar init.